viewer/web/daily.js

```diff
@@ -223,6 +238,8 @@ function submit_opinion() {
 
       // Report result
       set_flash(result.result, result.message)
+      // Update the total vote count
+      document.getElementById("opinion-count") = decision_count
```

Comment:
You need to set the whole thing.  Also, per the above changes, this should be read back from the server after update:
```suggestion
      document.getElementById("opinion-count").innerHTML = "# of existing votes: " + result.opinion_count
```